lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -163,7 +163,6 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
 
     HRESULT EnqueuePromiseTask(Js::Var taskVar) override
     {
-        AssertMsg(false, "jsrt should have set the promise callback");
```

Comment:
Please correct me if I'm wrong, but I recall that when I made a change to `nativetests` and was trying to test an error case that had a debug assert when the test case hit it, the assert caused the process to exit, and the test wasn't even marked as failed. My perspective here was that if we are explicitly supporting un-setting the promise callback, then having a promise complete with no completion handler is no longer assert-worthy, but instead a documented error case.